__tests__/app.test.js

```diff
@@ -207,6 +207,12 @@ describe("POST /api/articles/:article_id/comments", () => {
   });
 });
 
+describe("DELETE /api/comments/:comment_id", () => {
+  test("204 Successfully delete a comment by comment_id", () => {
+    return request(app).delete("/api/comments/1").expect(204);
+  });
```

Comment:
Nice first test, but there should be some error handling tests here as well:

When given an invalid comment id:
E.g `/api/comments/banana`

When given a valid but none existent comment id:
E.g `/api/comments/9999`